packages/paste-core/components/display-pill-group/stories/index.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {CalendarIcon} from '@twilio-paste/icons/esm/CalendarIcon';
+import {DisplayPillGroup} from '../src/DisplayPillGroup';
+import {DisplayPill} from '../src/DisplayPill';
+
+export const Basic: React.FC = () => {
+  return (
+    <DisplayPillGroup data-testid="display-pill-group" aria-label="Your favorite sports:">
+      <DisplayPill
+        data-testid="display-pill-anchor"
+        onFocus={() => {
+          console.log('Focused Tennis!');
+        }}
+        onBlur={() => {
+          console.log('Blurred Tennis!');
+        }}
+        href="https://google.com"
+      >
+        <CalendarIcon decorative size="sizeIcon10" />
+        Tennis
+      </DisplayPill>
+      <DisplayPill data-testid="display-pill-blank">Football</DisplayPill>
+      <DisplayPill href="/">Baseball</DisplayPill>
+      <DisplayPill>Basketball</DisplayPill>
+      <DisplayPill>Soccer</DisplayPill>
+    </DisplayPillGroup>
```

Comment:
Right now, the behavior I get when I tab through these Display Pill groups is that the focus skips over the Display Pills that don't have hrefs (from Tennis to Baseball, it skips Football, then skips the rest of the Display Pills on the next tab press). Is that what we want?

Reply:
That's correct. It only focuses over href pills.